### DISCUSSION TITLE
JSON Validator defines 'description' as required for in-network, schema says 'optional'

### DISCUSSION THREAD

**Discussion by olderthandirt5** [CMSgov/price-transparency-guide-validator#43] [Q&A] [answered]
Output from Validator:

> Input JSON is invalid.
> Error Name: required
> Message: Object is missing the following members required by the schema: 'description'.
> Instance: #/in_network/0
> Schema: #/definitions/in_network

https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#in-network-object

JSON tested:
  "in_network": [{ 
    "negotiation_arrangement" : "ffs", 
    "name" : "CLEAR EYELID GLAND W/HEAT", 
    "billing_code_type" : "CPT", 
    "billing_code_type_version" : "2020", 
    "billing_code" : "0207T", 
    "negotiated_rates" : [
        {
            "provider_references" : [
                1
            ], 
            "negotiated_prices" : [
                {
                    "negotiated_type" : "negotiated", 
                    "negotiated_rate" : 37.09, 
                    "expiration_date" : "9999-12-31", 
                    "service_code" : [
                        "11", 
                        "21", 
                        "22"
                    ], 
                    "billing_class" : "professional"
                }
            ]
        },

Adding a description to the first object allows it to pass, but the error shows up on the next object.

Is description required for in-network rates?

**Comment by mint-thompson** [comment]
This appears to be a discrepancy between the schema definition, which lists this field as required, and the schema's README file, which lists this field as optional. I suggest opening an issue on the price-transparency-guide repo so that this can be addressed.

**Reply by olderthandirt5** [reply]
Thanks. Will do.

**Comment by shaselton-usds** [comment, marked as answer]
@olderthandirt5 Thanks for pointing this out -- there was a PR that fixed the documentation issue a while back: https://github.com/CMSgov/price-transparency-guide/pull/442
